bundles/org.openhab.core.model.item/src/org/openhab/core/model/Items.xtext

```diff
@@ -17,9 +17,10 @@ ModelItem:
 	(ModelNormalItem | ModelGroupItem) name=ID
 	(label=STRING)?
 	('<' icon=Icon '>')?
-	('(' groups+=ID (',' groups+=ID)* ')')? 
-	('[' tags+=(ID|STRING) (',' tags+=(ID|STRING))* ']')?
-	('{' bindings+=ModelBinding (',' bindings+=ModelBinding)* '}')? 
+	('(' groups+=ID (',' groups+=ID)* ')')?
+	// Add BaseModelItemType as valid tags otherwise the parser treats [Switch] as start of item definition
+	('[' tags+=(ID|STRING|BaseModelItemType) (',' tags+=(ID|STRING|BaseModelItemType))* ']')?
+	('{' bindings+=ModelBinding (',' bindings+=ModelBinding)* '}')?
```

Comment:
Can you check what was changed at lines 20 and 22 ?
Did you replace spaces by tab or vice-versa ?

Reply:
Maybe a CRLF issue?

Reply:
They had a trailing space, my editor is set to automatically remove them on save.